lib/index.js

```diff
@@ -78,13 +124,19 @@ function create(options) {
 	};
 }
 
+/**
+ * @param {ResolveOptionsOptionalFS} options Resolver options
+ * @returns {{
+ *	(context: object, path: string, request: string): string|false;
+ *	(path: string, request: string): string|false;
+ * }} Resolver function
+ */
```

Comment:
Why do not move it to own type (same for other)?

Reply:
yeah i agree. this one is specific to the synchronous resolver but i can extract it into a typedef and do the same for the async one

Reply:
Yeah, let's do it